docs/source/getting_started.rst

```diff
@@ -0,0 +1,21 @@
+Getting Started
+===============
+
+The Data Resource Generator (DRG) is a database and API as configuration application.
+
+Terms
+-----
+
+Data Resource
+    These are things that can hold and handle data. In this case it is a database with a RESTful API
```

Comment:
Hmmmmm.....I do not like "things" as a descriptor. How about "Entity"? So, I'd rewrite it like so:

```
Data Resource: an entity that stores and handles data. In the context of the BrightHive DRG, a Data Resource is a relational database with a RESTful API.
```